src/containers/User/UserArea.tsx

```diff
@@ -182,17 +183,23 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({ templates, dataViews }) => {
 
 const BrandArea: React.FC = () => {
   const { strings } = useLanguageProvider()
+  const { preferences } = usePrefs()
```

Comment:
hmm, this potentially will cause a flicker between defaultBrandLogo and logo in prefs on slower networks.  

probably ok, but would typically wait for preferences in usePrefs before rendering Image.

Reply:
In "index.tsx", the entire app doesn't get rendered until prefs have loaded (it's always been this way, since we need the language strings before we can render anything). So by the time it gets to here, preferences will be available from the global "Prefs" context.

Reply:
ohh true, oops, i though prefs are re-fetched here, but that's not the case.